easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.1.1-fosscuda-2020b.eb

```diff
@@ -0,0 +1,189 @@
+easyblock = 'PythonBundle'
+
+name = 'AlphaFold'
+version = '2.1.1'
+
+homepage = 'https://deepmind.com/research/case-studies/alphafold'
+description = "AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known"
+
+toolchain = {'name': 'fosscuda', 'version': '2020b'}
+
+builddependencies = [
+    # CMake/Doxygen/SWIG are required for building OpenMM
+    ('CMake', '3.18.4'),
+    ('Doxygen', '1.8.20'),
+    # required for installing dm-tree
+    ('Bazel', '3.7.2'),
+    ('SWIG', '4.0.2'),
+]
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('PyYAML', '5.3.1'),
+    ('TensorFlow', '2.4.1'),
+    ('Biopython', '1.78'),
+    ('HH-suite', '3.3.0'),
+    ('HMMER', '3.3.2'),
+    ('Kalign', '3.3.1'),
+    ('jax', '0.2.19'),  # also provides absl-py
+]
+configopts = "-DCUDA_CUDA_LIBRARY=${EBROOTCUDA}/lib64/stubs/libcuda.so "
```

Comment:
@ThomasHoffmann77 This was also added to the easyconfig for AlphaFold 2.0.1 (it's not in the AlphaFold 2.0.0 easyconfig), but why is it needed, can you clarify (perhaps with a comment in the easyconfig itself above this line)?

Also, why is it not specific to a particular component? Like this, it applies to all components, is that intentional?

Reply:
Actually, that's not entirely correct: the `configopts` value is set to include this for all components, but `configopts` is totally ignored by the `PythonPackage` easyblock, so this actually only applies to the `OpenMM` component.
So, it would be better to specify this via `'configopts': ...` for the `OpenMM` component specifically, to make that clear (along with a comment to clarify why it is required at all).

Reply:
It is not generically needed. We didn't need it in our install. So if @ThomasHoffmann77 need it they should handled that in a hook.

Reply:
@boegel @akesandgren The line is a leftover from our configuration. I forgot to remove it. We require the line on our system, because we build on nodes missing the Nvidia drivers.